src/mame/palm/palm.cpp

```diff
@@ -17,6 +17,7 @@
 #include "machine/ram.h"
 #include "sound/dac.h"
 #include "video/sed1375.h"
+#include "video/mc68328lcd.h"
```

Comment:
Could you put these in alphabetical order?  There aren’t a huge number of headers right now, but when the number grows, stuff that isn’t in alphabetical order is a common source of duplicates or leftovers.

Reply:
Done